Command/AssetsInstallCommand.php

```diff
@@ -3,25 +3,23 @@
 namespace Admingenerator\GeneratorBundle\Command;
 
 use Admingenerator\GeneratorBundle\Filesystem\RelativePathComputer;
+use Symfony\Component\Console\Attribute\AsCommand;
 use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\OutputInterface;
 use Symfony\Component\Process\Process;
 
 /**
- * Class AssetsInstallCommand
  * @package Admingenerator\GeneratorBundle\Command
  * @author Stéphane Escandell
  *
  * Automatically call bower install to properly import bowers components.
  * Push them to the web root directory
  */
+#[AsCommand('admin:assets-install', 'Fetch bower declared dependencies and push them into web root directory')]
```

Comment:
Description needs to be in the configure, the name in the xml configuration where the tag is applied.

Reply:
And then we can still support 5.4